packages/react-dom/npm/client.react-server.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+throw new Error(
+  'react-dom/client is not supported in React Server Components.'
+);
```

Comment:
This should also be the same for `react-dom/server` since an SSR render consumes Client Components which don't exist in a react-server layer.